### PR TITLE
release info for 7.4

### PR DESCRIPTION
**Pull request by hkollmann**

### CODE REVIEW

File: docs/release-notes.7.0.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Qooxdoo Release Notes
 
+## New features in v7.4
+- compile.json: localModules can now be used in the root of the file. So you can adress the modules for all apllications.
```
Comment (derrell) [inline docs/release-notes.7.0.md:4]: s/adress/address/
s/apllications/applications/

File: docs/release-notes.7.0.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+                Application modules will be merged with global ones.
+- browserify of npm modules: Browserify of the used npm modules is a time cosuming process. No it's checked if it's really neccesary.
```
Comment (derrell) [inline docs/release-notes.7.0.md:6]: s/Browserify of the/Broserifying the/
s/time cosuming/time-consuming/
s/No it's chechecked if/Now it's checked whether/

File: docs/release-notes.7.0.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+It's neccesary when:
+    - commonjs-browserify.js do not exits
```
Comment (derrell) [inline docs/release-notes.7.0.md:8]: s/do not exits/does not exist/

File: docs/release-notes.7.0.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+    - The list of modules has changed
+    - One of the localModules are newer compared to the last run
```
Comment (derrell) [inline docs/release-notes.7.0.md:10]: s/One of/One or more of/

**Review by derrell — CHANGES_REQUESTED** [review]

File: docs/release-notes.7.0.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
 # Qooxdoo Release Notes
 
+## New features in v7.4
+- compile.json: localModules can now be used in the root of the file. So you can address the modules for all applications.
+                Application modules will be merged with global ones.
+- browserifying the npm modules: Browserify of the used npm modules is a time-consuming process. No it's checked whether it's really neccesary.
```
Comment (derrell) [inline docs/release-notes.7.0.md:6]: s/No it's checked/Now it's checked/

**Review by derrell — CHANGES_REQUESTED** [review]

**Review by derrell — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Pull request merged**